packages/angular-cli/models/config/config.ts

```
@@ -80,10 +80,16 @@ export class CliConfig<JsonType> {
 
     try {
       content = JSON.parse(configContent);
+    } catch (err) {
+      throw 'Parsing angular-cli.json failed. Please make sure your angular-cli.json is valid'
```

Can you throw a new `InvalidConfigError` with the message both here and below?